File: openapi-generator/dart-jaguar/lib/model/sensor_history_response_results.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'sensor_history_response_results.jser.dart';

class SensorHistoryResponseResults {
   /* List of datapoints, one for each requested (sensor, field) pair. */
  @Alias('series')
  final List<int> series;
   /* Timestamp in UNIX milliseconds. */
  @Alias('timeMs')
  final int timeMs;
  

  SensorHistoryResponseResults(
      

{
     this.series = const [],  
     this.timeMs = null 
    
    }
  );

  @override
  String toString() {
    return 'SensorHistoryResponseResults[series=$series, timeMs=$timeMs, ]';
  }
}

@GenSerializer(nullableFields: true)
class SensorHistoryResponseResultsSerializer extends Serializer<SensorHistoryResponseResults> with _$SensorHistoryResponseResultsSerializer {

}

File: openapi-generator/php-ze-ph/src/App/DTO/SensorHistoryResponseResults.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class SensorHistoryResponseResults
{
    /**
     * List of datapoints, one for each requested (sensor, field) pair.
     * @DTA\Data(field="series", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Type", "options":{"type":"int"}}
     * }})
     * @var int[]
     */
    public $series;
    /**
     * Timestamp in UNIX milliseconds.
     * @DTA\Data(field="timeMs", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $time_ms;
}
